frontend/src/components/pages/CampsPage.tsx

```diff
@@ -1,9 +1,85 @@
 import React, { useContext } from "react";
+import WaitlistedCampersTable from "./CampOverview/WaitlistedCampersTable";
```

Comment:
Just wanted to confirm that in the final version, we'll get the waitlistedCamper list from the camp overview page's get request right?